Core/Src/mqttsn_client_service.c

```diff
@@ -196,8 +229,8 @@ static void mqttsnServiceTask(EMBENET_TaskId taskId, EMBENET_NODE_TimeSource tim
             char uptimeStr[80];
             sprintf(uptimeStr, "{\"uptime\":%u}", (unsigned)(uptime / 1000U));
```

Comment:
either use snprintf or add a comment why you are sure about the resulting string always fitting uptimeStr